chainer/training/extensions/__init__.py

```diff
@@ -2,6 +2,7 @@
 from chainer.training.extensions import computational_graph  # NOQA
 from chainer.training.extensions import evaluator  # NOQA
 from chainer.training.extensions import exponential_shift  # NOQA
+from chainer.training.extensions import plot_report  # NOQA
```

Comment:
Thank you for changing the name. I think this name starts with "p", so it should be located one line before `print_report` at line 8.